src/components/breadcrumbs/_breadcrumb_content.styles.ts

```diff
@@ -25,6 +25,7 @@ export const euiBreadcrumbContentStyles = (euiThemeContext: UseEuiTheme) => {
   const { euiTheme } = euiThemeContext;
   return {
     euiBreadcrumb__content: css`
+      font-weight: ${euiTheme.font.weight.regular};
```

Comment:
We may want to add a comment here to avoid making the same mistake again in the future!
```suggestion
      /* Unset EuiLink's bolder font weight */
      font-weight: ${euiTheme.font.weight.regular};
```